### PR TITLE
Fix condition group checking issue at Throttle handler

### PR DESCRIPTION
**Pull request by ChamodDamitha**
Fix https://github.com/wso2/product-apim/issues/6432

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/throttling/ThrottleHandler.java [hunk 1]
```diff
@@ -275,43 +275,42 @@ private boolean doRoleBasedAccessThrottlingWithCEP(MessageContext synCtx, Config
...
-                        if (conditionGroupDTOs != null && conditionGroupDTOs.length > 0) {
-                            // Checking Applicability of Conditions is a relatively expensive operation. So we are
-                            // going to check it only if the API/Resource is throttled out.
-                            if (getThrottleDataHolder().isAPIThrottled
-                                    (resourceLevelThrottleKey)) {
-                                if (getThrottleDataHolder().isConditionsAvailable(resourceLevelThrottleKey)) {
-                                    Map<String, List<ConditionDto>> conditionDtoMap = getThrottleDataHolder()
-                                            .getConditionDtoMap(resourceLevelThrottleKey);
+                        if (getThrottleDataHolder().isAPIThrottled
+                                (resourceLevelThrottleKey)) {
```
Comment (TanyaM) [inline components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/throttling/ThrottleHandler.java:279]: can't this go in the same line 
`if (getThrottleDataHolder().isAPIThrottled (resourceLevelThrottleKey)) {`

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/throttling/ThrottleHandler.java [hunk 2]
```diff
@@ -275,43 +275,42 @@ private boolean doRoleBasedAccessThrottlingWithCEP(MessageContext synCtx, Config
...
+                                        long timestamp = getThrottleDataHolder().
+                                                getThrottleNextAccessTimestamp(combinedResourceLevelThrottleKey);
+                                        synCtx.setProperty(APIThrottleConstants.THROTTLED_NEXT_ACCESS_TIMESTAMP,
+                                                timestamp);
                                     }
-                                } else {
+                                }
+                            } else {
+                                if (conditionGroupDTOs != null && conditionGroupDTOs.length > 0) {
+
```
Comment (TanyaM) [inline components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/throttling/ThrottleHandler.java:313]: unnecessary new line?

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/throttling/ThrottleHandler.java (hunk 1 above)
Comment (ChamodDamitha) [reply components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/throttling/ThrottleHandler.java:279]: Fixed in https://github.com/wso2/carbon-apimgt/pull/9250

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/throttling/ThrottleHandler.java (hunk 2 above)
Comment (ChamodDamitha) [reply components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/throttling/ThrottleHandler.java:313]: Fixed in https://github.com/wso2/carbon-apimgt/pull/9250

**Pull request merged**